debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -313,6 +313,16 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue()
                     + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    public static final Field SNAPSHOT_SKIP_LOCKS = Field.create("snapshot.skip.locks")
```

Comment:
I'm not sure I necessarily agree with this configuration option.  What do others think?

Reply:
But I how can we take metadata locks on production DB. If table has 5M records and it will take it's own sweet time to snapshot but real TX failed because we are taking locks on it.  We introduced this as a configurable parameter

Reply:
@mallikankati I believe wht Gunnar is asking is that we consider using something similar to `SnapshotLockingMode` enum in the MySQL connector where we support `extended`, `minimal`, and `none`.  The idea here with an enum is that we can implement a variety of strategies rather than a simple lock or don't lock strategy.

Reply:
That make sense